spec/services/escort_creator_spec.rb

```diff
@@ -146,6 +164,15 @@ def expect_offences_to_be_cloned(existent_escort, new_escort)
     expect(offences_attributes).to match_array(expected_offences_attributes)
   end
 
+  def expect_move_special_vehicle_details_to_be_cloned(existent_escort, new_escort)
```

Comment:
Also for consistency, I would make this method similar to what we do for the cloned detainee:

```
  def expect_move_to_be_cloned(existent_escort, new_escort)
    move_attributes = existent_escort.move.attributes.except(*except_move_attributes)
    expect(new_escort.move.id).not_to eq(existent_escort.move.id)
    expect(new_escort.move).to have_attributes(move_attributes)
  end
```